app/javascript/service/views/Home.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    class="w-full h-full flex flex-col flex-no-wrap overflow-hidden bg-white"
+  >
+    <div class="flex flex-1 overflow-auto">
+      <div class="max-w-screen-sm w-full my-0 m-auto p-8">
+        <img src="/brand-assets/logo.svg" alt="Chatwoot logo" class="logo" />
```

Comment:
Didn't we actually agree on using the inbox avatar URL instead of Chatwoot logo?

Reply:
We will add this in API binding PR.